src/components/logo.jsx

```diff
@@ -84,7 +85,7 @@ export default class Logo extends Component {
 
 		return (
 			<svg
-				class="logo"
+				aria-label={title}
```

Comment:
`.logo` isn't used.

As for `aria-label`, it replaces a spread `title` prop (`<svg title="..."`) which does nothing to my knowledge, and is used instead of `<title>` for two reasons:

1. On hover, most browsers will show the content of `<title>` as a tooltip. We don't really need or want this in the places we use this component.
2. FireFox's accessibility tools show that `<title>` does not count as heading text in the scenario of `<h1><svg><title>...`, while `aria-label` does. This is important as we use this structure on our home page, where the main heading has no text in it, just our logo. With `<title>`, that heading is completely empty.